.rustfmt.toml

```diff
@@ -1,5 +1,2 @@
 edition = "2018"
-format_code_in_doc_comments = true
-format_macro_matchers = true
-merge_imports = true
```

Comment:
Is this really necessary? There are warning in stable but we can make nightly a requirement for formatting / developing.

Reply:
I'm not happy with it either but that's the price we have to pay for developing in stable.

I'm not sure if I like the idea of having nightly as a dev requirement. what if something runs in nightly but not in stable? of course CI will catch it but then you have to tell everyone to have nightly for formatting purposes and then switch to stable to run the tests locally and be sure the code will pass CI.

Reply:
> I'm not happy with it either but that's the price we have to pay for developing in stable.

I thought we just wanted it to compile / be usable on stable? And you can just use stable as default but for formatting use `cargo +nightly fmt`. I don't think it is that inconvenient.

Reply:
I'll add a comment to the contrib file